src/persistentStore.js

```diff
@@ -43,9 +43,24 @@ const store = new Store({ name: 'pc-nrfconnect-toolchain-manager' });
 export const isFirstInstall = () => store.get('isFirstInstall', true);
 export const setHasInstalledAnNcs = () => store.set('isFirstInstall', false);
 
-const defaultInstallDir = path.resolve(os.homedir(), 'ncs');
+const defaultInstallDir = {
+    win32: path.resolve(os.homedir(), 'ncs'),
+    darwin: '/opt/nordic/ncs',
+    linux: '//TODO',
+}[process.platform];
+
 export const persistedInstallDir = () => store.get('installDir', defaultInstallDir);
 export const setPersistedInstallDir = dir => store.set('installDir', dir);
 
-export const toolchainIndexUrl = () => store.get('toolchainIndexUrl', 'https://developer.nordicsemi.com/.pc-tools/toolchain/index.json');
+const defaultIndexJson = {
+    win32: 'index.json',
+    darwin: 'mac.json',
+    linux: 'linux.json',
+}[process.platform];
```

Comment:
From electron-updator's inspiration, maybe `index.json`, `index-mac.json` and `index-linux.json` are better.

Reply:
That's a good idea, I'll update the relevant scripts.